docs/proposed/http-storage-node-protocol.rst

```diff
@@ -252,11 +252,67 @@ For example::
       "delete-mutable-shares-with-zero-length-writev": true,
       "fills-holes-with-zero-bytes": true,
       "prevents-read-past-end-of-share-data": true,
-      "gbs-anonymous-storage-url": "pb://...#v=2"
+      "gbs-anonymous-storage-url": "pb://...#v=1"
       },
     "application-version": "1.13.0"
     }
 
+``PUT /v1/lease/:storage_index``
+!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+Create a new lease that applies to all shares for the given storage index.
+The details of the lease are encoded in the request body.
+For example::
+
+  {"renew-secret": "abcd", "cancel-secret": "efgh"}
```

Comment:
Is there a reason for these be encoded in the request body, rather than making them request parameters?  (As in, "`PUT /v1/lease/:storage_index&renew-secret=abcd&cancel-secret=efgh`".)

I am wondering if there's a case for writing an OpenAPI spec, because there's better tooling support, and there's less room for confusion.

Reply:
I think one reason (at least historically in Tahoe) is the idea that "URLs leak" and so if something along the way happens to log the URL you'll be including a secret in a random log-file.

Reply:
That makes sense.  If that is the case, could that reasoning also be documented?